src/app/modules/board/pages/board-page/board-page.component.html

```diff
@@ -1 +1,9 @@
-<app-header></app-header>
+<app-board-header></app-board-header>
+<div cdkDropListGroup class="board-columns" fxLayout="row" fxLayoutAlign="none">
+  <app-board-column
+    *ngFor="let column of columns$ | async"
+    [state]="column"
+    (taskDropped)="taskDropped($event)"
+    (taskClick)="taskClick($event)"
```

Comment:
taskClicked albo taskSelected, do nazewnictwa eventów stosujmy czas przeszły, a do funkcji które zostaną wykonane dodajmy prefix "on", czyli:

```suggestion
    (taskClicked)="onTaskClicked($event)"
```